### PR TITLE
Adds support for elavegan.com

### LINKED ISSUE #906: elavegan.com
Please check that recipes published on the website you're requesting are public (we can't currently scrape recipes that require an account login), and add sample recipe URL(s) below:

- https://elavegan.com/vegan-lentil-bolognese/
- https://elavegan.com/red-lentil-dahl/

### PR DESCRIPTION
**Pull request by jknndy**
Includes ingredient grouping support

Resolves #906

### CODE REVIEW

File: tests/test_elavegan_1.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                "1 tsp ground turmeric",
+                "1 tsp paprika",
+                "Sea salt and black pepper (to taste)",
+                "1/3 tsp red pepper flakes ((optional))",
+            ],
+            self.harvester_class.ingredients(),
+        )
+
+    def test_instructions(self):
+        self.assertEqual(
```
Comment (jayaddison) [inline tests/test_elavegan_1.py:66]: Nitpick: this could be more readable with the same `expected_instructions` list format used in the `_2` test module.

**Review by jayaddison — APPROVED** [review]
Looks great - thank you @jknndy.  I'll leave this open for a while in case you'd like to edit the `_1` instructions test; either way I'll merge this within a few days.

**Comment by jknndy** [comment]
Updated the instructions test case, should be ready for merge!

**Comment by jayaddison** [comment]
Thanks again @jknndy!  Merging...

**Pull request merged**